content/https/https-4/contents.lr

```diff
@@ -0,0 +1,11 @@
+_model: question
+---
+title: Why using DoH (DNS over HTTPS) is not enough to protect your privacy?
+---
+seo_slug: doh-privacy-protection
+---
+description:
+The short answer is: **No actual privacy is added by DoH.**

```

Comment:
"No actual privacy is added by DoH" is an overstatement, and something that will get this perspective dismissed as a hack job.

DoH provides real privacy *against your network provider*, but not against the resolver operator.